### ISSUE TITLE
no result

### ISSUE THREAD

**Issue by mama110** [eriklindernoren/PyTorch-YOLOv3#55]
I trained my own dataset. Now i use detect.py to detect the image, but i only find a big white bounding box around the image, there is  nothing else. Does anyone help? Thanks.

**Comment by libzzluo** [comment]
@mama110 @eriklindernoren  i met the same situation as you. And when i run the `test.py`, the output mAP is 0.0000(my training dataset is COCO and the eval is 5k.txt). It seems that detector can not load my own `.weights` file or just some mistakes. I trained the yolov3.cfg on COCO from scratch about 20 epochs, and i tried my another `.weights` file from redmon's origin Darknet implement, it worked. I guess that it may caused by the `.weights` saved by the author's code.

**Comment by ehosseiniasl** [comment]
@eriklindernoren model(x) in eval mode output all nan. I think save_weights is deprecated!
loading from `yolov3.weights` is fine!

**Comment by eriklindernoren** [comment]
Hi, let's keep training issues in #2.

**Comment by beckhamchen** [comment]
> @eriklindernoren model(x) in eval mode output all nan. I think save_weights is deprecated!
> loading from `yolov3.weights` is fine!

I also met this problem. Have you solved it?
